rebar.config

```diff
@@ -63,6 +63,7 @@
     , {mongodb, {git,"https://github.com/emqx/mongodb-erlang", {tag, "v3.0.13"}}}
     , {epgsql, {git, "https://github.com/emqx/epgsql.git", {tag, "4.6.0"}}}
     , {grpc, {git, "https://github.com/emqx/grpc-erl", {tag, "0.6.6"}}}
+    , {wolff, {git, "https://github.com/kafka4beam/wolff", {tag, "1.5.6"}}}
```

Comment:
this should be a different PR ?

Reply:
This is part of test commit and will be revert later, I just want to compare the relup file